### PR TITLE
A couple of changes to get the basic Plutus app compiling on the playground.

### PR DESCRIPTION
**Pull request by robertwb**
Especially someone brand new to the project, it's non-trivial to try and debug things like this when the very first tutorials fail with surprising compile errors. 

Pre-submit checklist:
- Branch
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [ ] Reviewer requested
- If you updated any cabal files or added Haskell packages:
    - [ ] `nix-shell shell.nix --run updateMaterialized` to update the materialized Nix files
    - [ ] Update `hie-*.yaml` files if needed
- If you changed any Haskell files:
    - [ ] `nix-shell shell.nix --run fix-stylish-haskell` to fix any formatting issues
- If you changed any Purescript files:
    - [ ] `nix-shell shell.nix --run fix-purty` to fix any formatting issues

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge

### CODE REVIEW

File: doc/tutorials/BasicApps.hs [hunk 1]
```diff
@@ -41,7 +41,7 @@ data SplitData =
     deriving stock (Show, Generic)
 
 -- For a 'real' application use 'makeIsDataIndexed' to ensure the output is stable over time
-PlutusTx.unstableMakeIsData ''SplitData
+PlutusTx.makeIsData ''SplitData
```
Comment (j-mueller) [inline doc/tutorials/BasicApps.hs:44]: `makeIsData` doesn't exist anymore, it's either `unstableMakeIsData` or `makeIsDataIndexed`, as the comment says.

Comment (robertwb) [reply doc/tutorials/BasicApps.hs:44]: Hmm... This doesn't reflect what I've been experiencing on https://playground.plutus.iohkdev.io/ (which seems to be the advised way to learn this kind of thing).

Comment (j-mueller) [reply doc/tutorials/BasicApps.hs:44]: the playground isn't running the latest version from master, it is only updated occasionally.

**Comment by robertwb** [comment]
Ack. It would be good for the tutorials to match the playground. Also, any
idea at what point the API will be stable?

**Comment by j-mueller** [comment]
Yeah it's a bit odd, the documentation that's published with the playground is newer than the Playground code itself, but it is not the `master` version either. So I think at some point after releasing the Playground the docs were published again for some reason.

The API should stabilisie over the next 2-4 weeks.

**Comment by michaelpj** [comment]
I think this is due to the deployed playground being old, we shouldn't change master.

**Pull request closed without merging**